### ISSUE TITLE
Extension never updates

### ISSUE THREAD

**Issue by timbertson** [lquixada/my-travis#32]
I installed it today, and configured it to update every 10 minutes. But it never updates the status - 3 hours after my latest build completed (I was online the whole time), it still showed "in progress".

Modifying the settings and clicking "save" triggers an immediate update, but the periodic ones seem to be broken. Is there a log somewhere I can check to see what's going on?

**Comment by lquixada** [comment]
Hi @gfxmonk! Thanks for the feedback! Although I'm still trying to figure out what could have happened, there's a temporary fix that you can try:
Go to Window > Extension and locate MyTravis there. Click Reload. See if that works for you.

**Comment by lquixada** [comment]
oops, still waiting for an answer :)

**Comment by timbertson** [comment]
Sorry, I should have given more info - I'm on fedora 18, chrome 25

I don't have a Window menu, but I assume it's the same as my [menu] -> Tools -> Extensions. There's no Reload button on that page, however.

Either way a manual reload button doesn't sound particularly useful.

**Comment by lquixada** [comment]
yeah, you're right! There's only a Reload link on local installs. :/ sorry!

I'm working on a fix right now! I'll let you know when it's ready!

**Comment by lquixada** [comment]
Ok, made some improvement. Can you update the extension? Now it's on v0.4.2.1.

**Comment by timbertson** [comment]
Thanks. I'm on 0.4.2.1 now, and have restarted chrome just in case.

I had a look at "chrome://net-internals" -> events and set the filter to "api.travis-ci.org". After updating the extension's settings I can see a bunch of stuff hitting api.travis-ci.org.

But I kept watching it for a few minutes after (I've set my update interval to 1), and nothing further hits that domain, so it looks like it's still not working.
